frontend/src/scenes/insights/views/LineGraph/LineGraph.tsx

```diff
@@ -1046,6 +1060,36 @@ export function LineGraph_({
         showMultipleYAxes,
         _goalLines,
         theme,
+        showTrendLines,
```

Comment:
anyone know why prettier pre-commit hook won't let me commit unless all of these dependencies are added? any way to get around this? I only need `showTrendLines` here